runtime/autoload/health/nvim.vim

```diff
@@ -29,6 +29,26 @@ function! s:check_config() abort
   endif
 endfunction
 
+function! s:check_vimrc() abort
+  let ok = v:true
+  call health#report_start('.vimrc File')
+
+  if !filereadable($HOME.'/.config/nvim/init.vim') && filereadable($HOME.'/.vimrc')
```

Comment:
The `.vimrc` file could either be at `~/.vimrc` *or* `~/.vim/vimrc`. Checking both locations would make sense, and maybe the advice that follows should also be adapted to the different locations.